src/Main.java

```diff
@@ -1,5 +1,100 @@
+import java.util.Random;
+
 public class Main {
     public static void main(String[] args) {
-
+        System.out.println("Задача 1");
+        // Объявите три массива:
+        // Целочисленный массив, заполненный тремя цифрами — 1, 2 и 3 — с помощью ключевого слова new
+        int i;
+        int[] number = new int[3];
+        number[0] = 1;
+        number[1] = 2;
+        number[2] = 3;
+        for(i = 0; i < number.length;i++){
+            System.out.println("Значение ячейки массива "+i+ ": " + number[i]);
+        }
+        System.out.println();
+        // Массив, в котором можно хранить три дробных числа — 1.57, 7.654, 9.986. Массив сразу заполните значениями.
+        float[] floatNumber = new float[3];
```

Comment:
здесь надо было сразу заполнить значениями:
```suggestion
        float[] floatNumber = {1.57f, 7.654f, 9.986f};
```